admin/controllers/service-manager.js

```diff
@@ -458,6 +460,71 @@ const controller = {
 
     req.flash('info', `'${school.name}' added as an MOD school`)
     return res.redirect(`/service-manager/mod-settings?hl=${school.urlSlug}`)
+  },
+
+  /**
+   * Manage service message
+   * @param req
+   * @param res
+   * @param next
+   * @returns {Promise.<void>}
+   */
+  getServiceMessage: async (req, res, next) => {
+    res.locals.pageTitle = 'Manage service message'
+    req.breadcrumbs(res.locals.pageTitle)
+    let serviceMessage
+    let redisResult
+    try {
+      const redisKey = 'serviceMessage'
+      redisResult = await redisCacheService.get(redisKey)
```

Comment:
the cache implementation should be managed by the service, not the controller.  you are spreading the implementation across 2 layers here.

Reply:
@GuyHarwood I was about to push a refactor on this when you commented.
Thought exactly the same thing. 😄